cfme/infrastructure/pxe.py

```diff
@@ -35,12 +35,12 @@
     fields=[
         ('name_text', Input('name')),
         ('log_protocol', Select("//select[@id='log_protocol']")),
-        ('uri_text', Input('uri')),
```

Comment:
We need to try to find the root cause of this, as unfortunately things may sometimes be filled in quicker than we hope. We are currently trying to work with dev on this.